js/package.json

```diff
@@ -36,12 +36,12 @@
     "ci:build:app": "NODE_ENV=production webpack --config webpack/app",
     "ci:build:lib": "NODE_ENV=production webpack --config webpack/libraries",
     "ci:build:dll": "NODE_ENV=production webpack --config webpack/vendor",
-    "ci:build:npm": "NODE_ENV=production webpack --config webpack/npm",
-    "ci:build:jsonrpc": "babel-node ./scripts/build-rpc-json.js --output .npmjs/jsonrpc",
+    "ci:build:npm": "NODE_ENV=production babel npm/$LIBRARY/src --out-dir npm/$LIBRARY/dist --ignore spec.js --source-maps",
+    "ci:build:jsonrpc": "babel-node ./scripts/build-rpc-json.js > npm/jsonrpc/index.json",
```

Comment:
Well in the script it hasn't been obvious before. I changed it to redirecting because it seems infinitely more flexible, e.g. when debugging, and follows unix principles.

Will change it if you insist. 😉